tests/AliasesTest.php

```diff
@@ -73,8 +107,8 @@ public function testSetTrimsTrailingSlashes(): void
             '@backward' => 'backward\\\\',
         ]);
 
-        $this->assertSame('forward', $aliases->get('@forward'));
-        $this->assertSame('backward', $aliases->get('@backward'));
```

Comment:
I'd keep this check too.

Reply:
This PR change this behavior: now if we get only alias, then path returns as is.